### PR TITLE
tags/*.tag: change to 2-space indentation #7352

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes partially #7352
All remaining files in tags/*.tag

### CODE REVIEW

**Comment by sujeet14108** [comment]
Ready for review :)

File: src/main/webapp/WEB-INF/tags/helpPage.tag [hunk 1]
```diff
@@ -4,44 +4,44 @@
...
+  <link rel="stylesheet" href="<%= FrontEndLibrary.BOOTSTRAP_THEME_CSS %>" type="text/css">
+  <link rel="stylesheet" href="/stylesheets/teammatesCommon.css" type="text/css">
+  <link rel="apple-touch-icon" href="apple-touch-icon.png">
+  <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
+  <!--[if lt IE 9]>
+    <script src="https://cdn.jsdelivr.net/html5shiv/3.7.3/html5shiv.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/respond/1.4.2/respond.min.js"></script>
+  <![endif]-->
 </head>
 <body style="padding-top: 0;">
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/helpPage.tag]: This should also be indented.

File: src/main/webapp/WEB-INF/tags/page.tag [hunk 2]
```diff
@@ -11,34 +11,34 @@
...
+  <link type="text/css" href="<%= FrontEndLibrary.BOOTSTRAP_CSS %>" rel="stylesheet">
+  <link type="text/css" href="<%= FrontEndLibrary.BOOTSTRAP_THEME_CSS %>" rel="stylesheet">
+  <link type="text/css" href="/stylesheets/teammatesCommon.css" rel="stylesheet">
+  <jsp:invoke fragment="cssIncludes" />
+  <!--[if lt IE 9]>
+    <script src="https://cdn.jsdelivr.net/html5shiv/3.7.3/html5shiv.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/respond/1.4.2/respond.min.js"></script>
+  <![endif]-->
 </head>
 <body>
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/page.tag]: This should also be indented.

File: src/main/webapp/WEB-INF/tags/staticPage.tag [hunk 3]
```diff
@@ -7,63 +7,63 @@
...
+  <meta name="keywords" content="Teammates, peer feedback, peer evaluation, student, team, project, free">
+  <meta name="description" content="TEAMMATES is an Online Peer Feedback System for student team projects. It is completely free to use. Get your students to evaluate their performance in team projects, and view reports and summaries of their feedback and evaluations.">
+  <link type="text/css" href="<%= FrontEndLibrary.BOOTSTRAP_CSS %>" rel="stylesheet">
+  <link type="text/css" href="<%= FrontEndLibrary.BOOTSTRAP_THEME_CSS %>" rel="stylesheet">
+  <link type="text/css" href="stylesheets/teammatesCommon.css" rel="stylesheet">
+  <link rel="shortcut icon" href="/favicon.png">
+  <link rel="apple-touch-icon" href="apple-touch-icon.png">
+  <title>TEAMMATES - Online Peer Feedback/Evaluation System for Student Team Projects</title>
 </head>
 <body>
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/staticPage.tag]: This should also be indented.

File: src/main/webapp/WEB-INF/tags/helpPage.tag [hunk 4]
```diff
@@ -4,44 +4,44 @@
 <!DOCTYPE html>
 <html>
 <head>
```
Comment (sujeet14108) [inline src/main/webapp/WEB-INF/tags/helpPage.tag]: Done

**Pull request merged**